hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/context/properties/GlobalStaticProperties.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2020-2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hedera.node.app.service.mono.context.properties;
+
+import static com.hedera.node.app.service.mono.context.properties.PropertyNames.WORKFLOWS_ENABLED;
+
+import com.hedera.node.app.service.mono.context.annotations.CompositeProps;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+@Singleton
+public class GlobalStaticProperties {
+    private final PropertySource properties;
```

Comment:
For static properties, we only need to inject `BootstrapProperties` (which takes into account the JAR's _src/main/resources/bootstrap.properties_ plus any overrides from _data/config/bootstrap.properties_.)

This `@CompositeProps` instance is only needed to get any additional (dynamic) overrides from `0.0.121`.